### DISCUSSION TITLE
.well-known responses?

### DISCUSSION THREAD

**Discussion by hbjydev** [remix-run/remix#10152] [Q&A] [answered]
Hi all! I run a Matrix homeserver & connect my Bluesky account to my domain primarily through `.well-known/` files; is there an easy way to add these to a Remix site? :)

They're just text responses, so I figured having a `loader` that returns a `Response` object should work, just wondering if this is:
- A: the right way to do it
- B: something with examples I can go read

Thanks! :)

**Comment by sergiodxa** [comment, marked as answer]
You could add them to your `public` folder, if you need a specific content-type you can use a resource route, e.g. create a route `app/routes/[.]well-known.apple-app-site-association.ts` (note the `[.]` will escape the dot), and add a loader that sends the JSON response.
